workers/unity/Assets/Playground/Scripts/Player/ProcessLaunchCommandSystem.cs

```diff
@@ -62,7 +64,8 @@ protected override void OnUpdate()
                     {
                         ImpactPoint = info.ImpactPoint,
                         LaunchDirection = info.LaunchDirection,
-                        LaunchEnergy = energy
+                        LaunchEnergy = energy,
+                        Launcher = info.Launcher
```

Comment:
Not a bug introduced in this PR but it looks like this is missing the tag for authority on the injection group.